ddtrace/contrib/dbapi.py

```diff
@@ -103,19 +102,9 @@ def _trace_method(self, method, name, resource, extra_tags, dbm_propagator, *arg
             s.set_tag_str(SPAN_KIND, SpanKind.CLIENT)
 
             if asm_config._iast_enabled:
-                try:
```

Comment:
Can we avoid defining iast specific logic in the dbapi integration. We should use the core api instead